controllers/provisioners/eks/cloud.go

```diff
@@ -187,6 +206,24 @@ func (d *DiscoveredState) GetScalingGroup() *autoscaling.Group {
 	}
 	return &autoscaling.Group{}
 }
+
+// Included. Temporarily marked for traceability purposes: labelbug fix @agaro
+//######################################################################################
+func (d *DiscoveredState) SetCluster(cluster *eks.Cluster) {
+	d.Cluster = cluster
+}
+
+func (d *DiscoveredState) GetClusterVersion() (*semver.Version, error) {
+
+	ver, err := semver.NewVersion(*d.Cluster.Version)
```

Comment:
You can probably use the utility `aws.StringValue` to have protection against nil pointer.
e.g.
```
version := aws.StringValue(d.Cluster.Version)
ver, err := semver.NewVersion(version)
...
```

Reply:
Actually, can we also change this getter to only return `aws.StringValue(d.Cluster.Version)` and not the semver?
That way this getter does not need to return an error and we can handle the semver inside `GetLabelList`